components/dynatrace_activegate/nonprod-private.tf

```diff
@@ -1,11 +1,18 @@
 # For private ActiveGate synthethic monitoring
 module "nonprod_dynatrace_activegate_private" {
+
+  providers = {
+    azurerm     = azurerm
+    azurerm.law = azurerm.law_nonprod
+  }
+
   source                  = "../../modules/dynatrace-activegate"
   instance_count          = 2
   dynatrace_instance_name = "yrk32651"
   env                     = "nonprod"
   network_zone            = "azure.cft"
   config_file_name        = "cloudconfig-private"
+  enable_log_analytics    = true
```

Comment:
are you going to enable this for the other ones as well? was there a particular driver for this PR btw?

Reply:
Yea @timja most likely going forward. 

@naikajah raised the request (I think from pentest), wanted just to apply to the private active gate first.